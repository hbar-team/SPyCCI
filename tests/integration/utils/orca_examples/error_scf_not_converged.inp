%pal
  nprocs 8
end

%maxcore 750

! PBE def2-SVP 
! RIJCOSX def2/J


%scf
  maxiter 1
end

* xyzfile 0 1 water.xyz

