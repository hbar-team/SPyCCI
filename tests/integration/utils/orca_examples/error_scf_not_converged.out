
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ,####,    ,#,          
            ##       ##  ##   ,#'  ##            #'  '#       #'       ,# #          
            ##       ##  #######   ##           ,######,      #####,      #          
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,     #,   #      #          
             '#######'   ##     ##  '#######'  #'      '#     '####' #    #          



                #########################################################
                #                        -***-                          #
                #          Department of theory and spectroscopy        #
                #                                                       #
                #                      Frank Neese                      #
                #                                                       #
                #     Directorship, Architecture, Infrastructure        #
                #                    SHARK, DRIVERS                     #
                #        Core code/Algorithms in most modules           #
                #                                                       #
                #        Max Planck Institute fuer Kohlenforschung      #
                #                Kaiser Wilhelm Platz 1                 #
                #                 D-45470 Muelheim/Ruhr                 #
                #                      Germany                          #
                #                                                       #
                #                  All rights reserved                  #
                #                        -***-                          #
                #########################################################


                         Program Version 6.1.0  -  RELEASE   -
                                (GIT: $679e74b$)
                          ($2025-06-10 18:02:51 +0200$)


 With contributions from (in alphabetic order):
[Max-Planck-Institut fuer Kohlenforschung]                               
  Daniel Aravena         : Magnetic Suceptibility                        
  Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
  Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum      
  Ute Becker             : All parallelization in ORCA, NUMFREQ, NUMCALC 
  Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD           
  Dmytro Bykov           : pre 5.0 version of the SCF Hessian            
  Marcos Casanova-PÃ¡ez   : Triplet and SCS-CIS(D). UHF-(DLPNO)-IP/EA/STEOM-CCSD. UHF-CVS-IP/STEOM-CCSD   
  Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE 
  Pauline Colinet        : FMM embedding                                 
  Dipayan Datta          : RHF DLPNO-CCSD density                        
  Achintya Kumar Dutta   : EOM-CC, STEOM-CC                              
  Nicolas Foglia         : Exact transition moments, OPA infrastructure, MCD improvements    
  Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI      
  Miquel Garcia-Rates    : C-PCM and meta-GGA Hessian, CCSD/C-PCM, Gaussian charge scheme 
  Tiago L. C. Gouveia    : GS-ROHF, GS-ROCIS                             
  Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization        
  Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods 
  Ingolf Harden          : AUTO-CI MPn and infrastructure                
  Benjamin Helmich-Paris : MC-RPA, TRAH-(SCF,CASSCF), AVAS, COSX integrals, SCF dyn. polar., MC-PDFT, srDFT 
  Lee Huntington         : MR-EOM, pCC                                   
  Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM     
  Riya Kayal             : Wick's Theorem for AUTO-CI, AUTO-CI UHF-CCSDT 
  Emily Kempfer          : AUTO-CI RHF CISDT and CCSDT, approximate NEVPT4
  Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K, improved NEVPT2 
  Axel Koslowski         : Symmetry handling                             
  Simone Kossmann        : meta-GGA functionals, TD-DFT gradient, OOMP2, (MP2 Hessian; deprecated post 5.0) 
  Lucas Lang             : DCDCAS, Hyperfine gauge corrections, ICE-SOC+SSC
  Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC        
  Spencer Leger          : CASSCF response                               
  Dagmar Lenk            : GEPOL surface, SMD, ORCA-2-JSON               
  Dimitrios Liakos       : Extrapolation schemes; Compound Job, Property file
  Dimitrios Manganas     : Further ROCIS development; embedding schemes. LFT, Crystal Embedding 
  Dimitrios Pantazis     : SARC Basis sets                               
  Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients 
  Taras Petrenko         : pre 6.0 DFT Hessian and TD-DFT gradient,  ECA, NRVS 
  Petra Pikulova         : Analytic Raman intensities                    
  Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient                 
  Shashank Vittal Rao    : ES-AILFT, MagRelax                            
  Christoph Reimann      : Effective Core Potentials                     
  Marius Retegan         : Local ZFS, SOC                                
  Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples 
  Michael Roemelt        : Original ROCIS implementation, recursive CI coupling coefficients
  Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density      
  Barbara Sandhoefer     : DKH picture change effects                    
  Yorick L. A. Schmerwitz: GMF and freeze-and-release deltaSCF, NEB S-IDPP initial path                    
  Kantharuban Sivalingam : CASSCF convergence/infrastructure, NEVPT2, NEVPT3, NEVPT4(SD), FIC-MRCI and CEPA variants
  Bernardo de Souza      : ESD, SOC TD-DFT                               
  Georgi L. Stoychev     : AutoAux, RI-MP2 NMR, DLPNO-MP2 response, X2C  
  Van Anh Tran           : RI-MP2 g-tensors                              
  Willem Van den Heuvel  : Paramagnetic NMR                              
  Zikuan Wang            : NOTCH, Electric field optimization            
  Frank Wennmohs         : Technical directorship and infrastructure     
  Hang Xu                : AUTO-CI-Response properties                   
                                                                         
[FACCTs GmbH]                                                            
  Markus Bursch, Nicolas Foglia, Miquel Garcia-Rates, Ingolf Harden, Hagen Neugebauer, Anastasios Papadopoulos, 
  Christoph Riplinger, Bernardo de Souza, Georgi L. Stoychev                                                    
                                                                 
  APM, various basis sets, CI-OPT, improved COSX, DLPNO-Multilevel,           
  DOCKER, DRACO, updates on ESD, Fragmentator, GOAT, IRC, LR-CPCM, L-BFGS, MBIS, meta-GGA TD-DFT gradient, ML-optimized integration grids, 
  MM, NACMEs, nearIR, NEB, NEB-TS, NL-DFT gradient (VV10), 2- and 3-layer-ONIOM, interface openCOSMO-RS, QMMM,   
  Crystal-QMMM, RESP, rigid body optimization, SF, symmetry and pop. for TD-DFT, various functionals, SOLVATOR                                   
                                                                         
[Other institutions]                                                     
  V. Asgeirsson          : NEB                                           
  Christoph Bannwarth    : sTDA-DFT, sTD-DFT, PBEh-3c, B97-3c, D3        
  Giovanni Bistoni       : ETS/NOCV, ADLD/ADEX, COVALED                  
  Martin Brehm           : Molecular dynamics                            
  Ronald Cardenas        : ETS/NOCV                                      
  Martina Colucci        : COVALED                                       
  Sebastian Ehlert       : rSCAN, r2SCAN, r2SCAN-3c, D4, dhf basis sets  
  Marvin Friede          : D4 for Fr, Ra, Ac-Lr                          
  Lars Goerigk           : TD-DFT with DH, B97 family of functionals     
  Stefan Grimme          : VdW corrections, initial TS optimization, DFT functionals, gCP, sTDA/sTD-DF 
  Waldemar Hujo          : DFT-NL                                        
  H. Jonsson             : NEB                                           
  Holger Kruse           : gCP                                           
  Marcel Mueller         : wB97X-3c, vDZP basis set                      
  Hagen Neugebauer       : wr2SCAN, Native XTB                           
  Gianluca Regni         : ADLD/ADEX                                     
  Tobias Risthaus        : pre 6.0 range-separated hybrid DFT and stability analysis
  Lukas Wittmann         : regularized MP2, r2SCAN double-hybrids, wr2SCAN        
                                                                         
We gratefully acknowledge several colleagues who have allowed us to      
interface, adapt or use parts of their codes:                            
  Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods 
  Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG                   
  Ulf Ekstrom                                   : XCFun DFT Library      
  Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
  Frank Weinhold                                : gennbo (NPA and NBO analysis)           
  Simon Mueller                                 : openCOSMO-RS                            
  Christopher J. Cramer and Donald G. Truhlar   : smd solvation model                     
  S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library                           
  Liviu Ungur et al                             : ANISO software                          


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 7.0.0
 For citations please refer to: https://libxc.gitlab.io

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices
   BLAS/LAPACK       :  OpenBLAS 0.3.29  USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell SINGLE_THREADED
        Core in use  :  Haswell
   Copyright (c) 2011-2014, The OpenBLAS Project


  ***********************************
  * Starting time: Fri Sep 26 20:30:04 2025
  * Host name:     Host-001
  * Process ID:    34888
  * Working dir.:  /home/ppravatto/Documenti/Esperimenti/ChimicaComputazionale/SPyCCI/water_yxuds91x_orca_PBE_def2-SVP_vacuum_spe
  ***********************************

NOTE: MaxCore=750 MB was set to SCF,MP2,MDCI,CIPSI,MRCI,RASCI and CIS
      => If you want to overwrite this, your respective input block should be placed after the MaxCore statement


***************************************
The coordinates will be read from file: water.xyz
***************************************


================================================================================

----- Orbital basis set information -----
Your calculation utilizes the basis: def2-SVP
   F. Weigend and R. Ahlrichs, Phys. Chem. Chem. Phys. 7, 3297 (2005).

----- AuxJ basis set information -----
Your calculation utilizes the auxiliary basis: def2/J
   H-Rn: F. Weigend, Phys. Chem. Chem. Phys. 8, 1057 (2006).
   Fr-Lr: K. Eichkorn, F. Weigend, O. Treutler, R. Ahlrichs; Theor. Chem. Acc. 97, 119 (1997).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================



WARNING: Found flag for RIJCOSX approximation but no HF exchange needed!
  ===> : RIJCOSX  will be disabled and Split-RI-J is turned on

================================================================================
                                       INPUT FILE
================================================================================
NAME = input.inp
|  1> %pal
|  2>   nprocs 8
|  3> end
|  4> 
|  5> %maxcore 750
|  6> 
|  7> ! PBE def2-SVP 
|  8> ! RIJCOSX def2/J
|  9> 
| 10> 
| 11> %scf
| 12>   maxiter 1
| 13> end
| 14> 
| 15> * xyzfile 0 1 water.xyz
| 16> 
| 17> 
| 18>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  O     -3.210350   -0.585040   -0.013950
  H     -2.242470   -0.618270    0.018480
  H     -3.489200   -1.249110    0.634290

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 O     8.0000    0    15.999   -6.066682   -1.105565   -0.026362
   1 H     1.0000    0     1.008   -4.237654   -1.168361    0.034922
   2 H     1.0000    0     1.008   -6.593632   -2.360476    1.198634

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     0.968993102246     0.00000000     0.00000000
 H      1   2   0     0.969000198658   103.97800340     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     1.831131588903     0.00000000     0.00000000
 H      1   2   0     1.831144999180   103.97800340     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 2 groups of distinct atoms

 Group   1 Type O   : 7s4p1d contracted to 3s2p1d pattern {511/31/1}
 Group   2 Type H   : 4s1p contracted to 2s1p pattern {31/1}

Atom   0O    basis set group =>   1
Atom   1H    basis set group =>   2
Atom   2H    basis set group =>   2
---------------------------------
AUXILIARY/J BASIS SET INFORMATION
---------------------------------
There are 2 groups of distinct atoms

 Group   1 Type O   : 12s5p4d2f1g contracted to 6s4p3d1f1g pattern {711111/2111/211/2/1}
 Group   2 Type H   : 5s2p1d contracted to 3s1p1d pattern {311/2/1}

Atom   0O    basis set group =>   1
Atom   1H    basis set group =>   2
Atom   2H    basis set group =>   2


           ************************************************************
           *        Program running with 8 parallel MPI-processes     *
           *              working on a common directory               *
           ************************************************************
------------------------------------------------------------------------------
                             ORCA STARTUP CALCULATIONS
                           -- RI-GTO INTEGRALS CHOSEN --
------------------------------------------------------------------------------
------------------------------------------------------------------------------
                   ___                                                        
                  /   \      - P O W E R E D   B Y -                         
                 /     \                                                     
                 |  |  |   _    _      __       _____    __    __             
                 |  |  |  | |  | |    /  \     |  _  \  |  |  /  |          
                  \  \/   | |  | |   /    \    | | | |  |  | /  /          
                 / \  \   | |__| |  /  /\  \   | |_| |  |  |/  /          
                |  |  |   |  __  | /  /__\  \  |    /   |      \           
                |  |  |   | |  | | |   __   |  |    \   |  |\   \          
                \     /   | |  | | |  |  |  |  | |\  \  |  | \   \       
                 \___/    |_|  |_| |__|  |__|  |_| \__\ |__|  \__/        
                                                                              
                      - O R C A' S   B I G   F R I E N D -                    
                                      &                                       
                       - I N T E G R A L  F E E D E R -                       
                                                                              
 v1 FN, 2020, v2 2021, v3 2022-2024                                           
------------------------------------------------------------------------------


----------------------
SHARK INTEGRAL PACKAGE
----------------------

Number of atoms                             ...      3
Number of basis functions                   ...     24
Number of shells                            ...     12
Maximum angular momentum                    ...      2
Integral batch strategy                     ... SHARK/LIBINT Hybrid
RI-J (if used) integral strategy            ... SPLIT-RIJ (Revised 2003 algorithm where possible)
Printlevel                                  ...      1
Contraction scheme used                     ... SEGMENTED contraction
Prescreening option                         ... SCHWARTZ
   Thresh                                   ... 1.000e-10
   Tcut                                     ... 1.000e-11
   Tpresel                                  ... 1.000e-11 
Coulomb Range Separation                    ... NOT USED
Exchange Range Separation                   ... NOT USED
Multipole approximations                    ... NOT USED
Finite Nucleus Model                        ... NOT USED
CABS basis                                  ... NOT available
Auxiliary Coulomb fitting basis             ... AVAILABLE
   # of basis functions in Aux-J            ...     71
   # of shells in Aux-J                     ...     25
   Maximum angular momentum in Aux-J        ...      4
Auxiliary J/K fitting basis                 ... NOT available
Auxiliary Correlation fitting basis         ... NOT available
Auxiliary 'external' fitting basis          ... NOT available

Checking pre-screening integrals            ... done (  0.0 sec) Dimension = 12
   => SHARK Basis and OBASIS are compatible. Storing Pre-screening
Shell pair information
Shell pair cut-off parameter TPreSel        ...   1.0e-11
Total number of shell pairs                 ...        78
Shell pairs after pre-screening             ...        78
Total number of primitive shell pairs       ...       272
Primitive shell pairs kept                  ...       265
          la=0 lb=0:     28 shell pairs
          la=1 lb=0:     28 shell pairs
          la=1 lb=1:     10 shell pairs
          la=2 lb=0:      7 shell pairs
          la=2 lb=1:      4 shell pairs
          la=2 lb=2:      1 shell pairs

Checking whether 4 symmetric matrices of dimension 24 fit in memory
:Max Core in MB      =    750.00
 MB in use           =      3.23
 MB left             =    746.77
 MB needed           =      0.01
 Data fit in memory  = YES
Calculating RI/J V-Matrix + Cholesky decomp.... done (  0.3 sec)
Calculating Nuclear repulsion               ... done (  0.0 sec) ENN=      9.084297099237 Eh

Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 3.830e-02
Time for diagonalization                   ...    0.026 sec
Threshold for overlap eigenvalues          ... 1.000e-07
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.113 sec
Total time needed                          ...    0.416 sec

-------------------
DFT GRID GENERATION
-------------------

General Integration Accuracy     IntAcc      ... 4.388
Radial Grid Type                 RadialGrid  ... OptM3 with GC (2021)
Angular Grid (max. ang.)         AngularGrid ... 4 (Lebedev-302)
Angular grid pruning method      GridPruning ... 4 (adaptive)
Weight generation scheme         WeightScheme... mBecke (2022)
Basis function cutoff            BFCut       ... 1.0000e-10
Integration weight cutoff        WCut        ... 1.0000e-14
Partially contracted basis set               ... off
Rotationally invariant grid construction     ... off
Angular grids for H and He will be reduced by one unit

Total number of grid points                  ...    12750
Total number of batches                      ...      201
Average number of points per batch           ...       63
Average number of grid points per atom       ...     4250
Grids setup in   1.2 sec
Initializing property integral containers    ... done (  0.4 sec)

SHARK setup successfully completed in   3.1 seconds

Maximum memory used throughout the entire STARTUP-calculation: 8.8 MB


           ************************************************************
           *        Program running with 8 parallel MPI-processes     *
           *              working on a common directory               *
           ************************************************************
-------------------------------------------------------------------------------
                                 ORCA GUESS
                   Start orbitals & Density for SCF / CASSCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Density Functional     Method          .... DFT(GTOs)
 Exchange Functional    Exchange        .... PBE
   PBE kappa parameter   XKappa         ....  0.804000
   PBE mue parameter    XMuePBE         ....  0.219520
 Correlation Functional Correlation     .... PBE
   PBE beta parameter  CBetaPBE         ....  0.066725
 LDA part of GGA corr.  LDAOpt          .... PW91-LDA
 Gradients option       PostSCFGGA      .... off
   NL short-range parameter             ....  6.400000
 RI-approximation to the Coulomb term is turned on
   Number of AuxJ basis functions       .... 71


General Settings:
 Integral files         IntName         .... input
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   10
 Basis Dimension        Dim             ....   24
 Nuclear Repulsion      ENuc            ....      9.0842970992 Eh

Convergence Acceleration:
 AO-DIIS                CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 MO-DIIS                CNVKDIIS        .... off
 Trust-Rad. Augm. Hess. CNVTRAH         .... auto
   Auto Start mean grad. ratio tolernc. ....  1.125000
   Auto Start start iteration           ....    50
   Auto Start num. interpolation iter.  ....    10
   Max. Number of Micro iterations      ....    24
   Max. Number of Macro iterations      .... Maxiter - #DIIS iter
   Number of Davidson start vectors     ....     2
   Converg. threshold    (grad. norm)   ....   5.000e-05
   Grad. Scal. Fac. for Micro threshold ....   0.100
   Minimum threshold for Micro iter.    ....   1.000e-02
   NR start threshold (gradient norm)   ....   1.000e-04
   Initial trust radius                 ....   0.400
   Minimum AH scaling param. (alpha)    ....   1.000
   Maximum AH scaling param. (alpha)    .... 1000.000
   Quad. conv. algorithm                .... NR
   White noise on init. David. guess    .... on
   Maximum white noise                  ....   0.010
   Pseudo random numbers                .... off
   Inactive MOs                         .... canonical
   Orbital update algorithm             .... Taylor
   Preconditioner                       .... Diag
   Full preconditioner red. dimension   ....   250
 SOSCF                 CNVSOSCF           .... on
   Start iteration     SOSCFMaxIt         ....   150
   Startup grad/error  SOSCFStart         ....  0.003300
   Hessian update      SOSCFHessUp        .... L-BFGS
   Autom. constraints  SOSCFAutoConstrain .... off
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000

SCF Procedure:
 Maximum # iterations   MaxIter         ....     1
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... SHARK and LIBINT hybrid scheme
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06

------------------------------
INITIAL GUESS: MODEL POTENTIAL
------------------------------
Loading Hartree-Fock densities                     ... done
Calculating cut-offs                               ... done
Initializing the effective Hamiltonian             ... done
Setting up the integral package (SHARK)            ... done
Starting the Coulomb interaction                   ... done (   0.6 sec)
Making the grid                                    ... done (   5.4 sec)
Mapping shells                                     ... done
Starting the XC term evaluation                    ... done (   0.1 sec)
  promolecular density results
     # of electrons  =      9.998906425
     EX              =     -8.726226855
     EC              =     -0.315781802
     EX+EC           =     -9.042008657
Transforming the Hamiltonian                       ... done (   0.8 sec)
Diagonalizing the Hamiltonian                      ... done (   0.7 sec)
Back transforming the eigenvectors                 ... done (   0.3 sec)
Now organizing SCF variables                       ... done
                      ------------------
                      INITIAL GUESS DONE (   9.3 sec)
                      ------------------
             **** ENERGY FILE WAS UPDATED (input.en.tmp) ****
Finished Guess after  11.6 sec
Maximum memory used throughout the entire GUESS-calculation: 5.4 MB


           ************************************************************
           *        Program running with 8 parallel MPI-processes     *
           *              working on a common directory               *
           ************************************************************

-------------------------------------------------------------------------------------------
                                      ORCA LEAN-SCF
                              memory conserving SCF solver
-------------------------------------------------------------------------------------------

----------------------------------------D-I-I-S--------------------------------------------
Iteration    Energy (Eh)           Delta-E    RMSDP     MaxDP     DIISErr   Damp  Time(sec)
-------------------------------------------------------------------------------------------
               ***  Starting incremental Fock matrix formation  ***
    1    -76.1856336716830640     0.00e+00  4.59e-02  8.89e-02  2.85e-01  0.700  13.9
[file orca_leanscf/orca_leanscf.cpp, line 306, Process 4]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 

               *****************************************************
               *                      ERROR                        *
               *        SCF NOT CONVERGED AFTER   0 CYCLES         *
               *****************************************************

[file orca_leanscf/orca_leanscf.cpp, line 306, Process 0]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 
[file orca_leanscf/orca_leanscf.cpp, line 306, Process 2]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 
[file orca_leanscf/orca_leanscf.cpp, line 306, Process 6]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 
[file orca_leanscf/orca_leanscf.cpp, line 306, Process 1]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 
[file orca_leanscf/orca_leanscf.cpp, line 306, Process 3]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 
[file orca_leanscf/orca_leanscf.cpp, line 306, Process 5]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 
[file orca_leanscf/orca_leanscf.cpp, line 306, Process 7]: Error (ORCA_LEANSCF): unfortunately, the SCF has not converged. There may be a way out but we have to stop here 

ORCA finished by error termination in LEANSCF
Calling Command: mpirun -np 8 --bind-to none --use-hwthread-cpus /home/storage/SOFTWARE/orca_6_1_0_linux_x86-64_shared_openmpi418/orca_leanscf_mpi input.gbw input 
[file orca_tools/qcmsg.cpp, line 394]: 
  .... aborting the run

